.azure-pipelines/testing-template.yml

```diff
@@ -1,30 +1,35 @@
 jobs:
-- ${{ each gids in parameters.gpu_inds }}:
-  - ${{ each config in parameters.configs }}:
+  - ${{ each dom in parameters.domains }}:
     - job:
-      displayName: "domain ${{config}} with GPUs ${{gids}}"
+      displayName: "domain ${{dom}} with 2 GPU"
```

Comment:
So we'll always have 2 GPUs? Or is there a chance that we can have 4? :D Maybe we can avoid hard-coding here? Just curious. :) You know the best :)

Reply:
The machine even before, always had 2 GPU :thinking:

Reply:
We don't want 4 here. We only want two to be able to test multi-gpu. Having more gpus would reduce the number of parallel workers which would reduce the throughput as these gpus are not under heavy load and thus increase the queues